### ISSUE TITLE
gettext does not extract some lines from a table

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1477]
gettext does not extract some lines from a table.

Create the following table for example:

```
+--------+----------------+--------------+
| Type   | Message        |  Comment     |
+========+================+==============+
| err    | | **Title 1**  | Comment1     |
|        | | Message 1    | cmd1         |
+--------+----------------+--------------+
| err    | | **Title 2**  | Comment2     |
|        | | Message 2    | cmd2         |
+--------+----------------+--------------+

```

gettext does not extract the following lines:

```
| **Title 1**
| Message 1  

| **Title 2**
| Message 2  
```

I think the problem in `|` instruction.
It's critical for me. I cannot create full translations for my documentation.

Please, find attached test project.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1477
- Originally reported by: Evgeny Prilepin
- Originally created at: 2014-05-28T16:35:12.184

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-05-28 16:26:21+00:00_

Confirmed. I'll fix it.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-05-28 16:29:43+00:00_

docutils doesn't set source/line information for line node in a table element. it's a reason of this issue. I'll fix it with small workaround.

**Comment by shimizukawa** [comment]
_From Evgeny Prilepin on 2014-05-30 12:44:09+00:00_

Thanks for your attention!

Also gettext does not extract strings from a similar list. For example:

```
* | Error message
  | |error-message-en|
* | Warning message
  | |warn-message-en|
* | Info message
  | |info-message-en|
```

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-08-25 16:40:30+00:00_
- gettext does not extract nodes.line in a table or list. Closes #1477

→ <<cset 8f7936d75cd39bfb3f7b4e54a0cb3705b962365d>>
